More_types/23Maps.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+// picパッケージ、stringsパッケージを取得
+import (
+	"strings"
+
+	"golang.org/x/tour/wc"
+)
+
+/**
+* WordCount関数を定義
+* 引数はstring型のsとし、返り値はキーはstring型、値はint型のmapを返す。
+ */
+func WordCount(s string) map[string]int {
+
+	/**
+	 * strings.Fields関数により、引数で渡された文字列を空白文字ごとに分割し、スライスとして返す
+	 * 空白のみが含まれる場合は空のスライスを返す
+	 */
+	word_list := strings.Fields(s)
+
+	// mapを作成する。キーはstring型、要素はint型を指定。
+	words := make(map[string]int)
```

Comment:
len(word_list)で、最大キャパシティが確定できるはずです。

```suggestion
	words := make(map[string]int, len(word_list))
```

メモリ効率が悪いでの、容量がわかっているところは決めていくようにしましょう。

Reply:
メモリ効率に影響が出るのですね。意識して書くようにします。